packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -210,14 +220,12 @@ export const useSliderState = (state: Pick<SliderState, keyof SliderCommon | key
   }
 
   // Track Props
-  state.track.className = 'ms-Slider-track';
   state.track.style = trackStyles;
 
   // Thumb Wrapper Props
   state.thumbWrapper.style = thumbStyles;
 
   // Thumb Props
-  state.thumb.className = 'ms-Slider-thumb';
```

Comment:
If you still need these (which I don't think you do looking at this implementation) you should add them back in `useSliderStyles` similar to what we discussed for `Switch`.

Reply:
Resolving as this was fixed in another PR